### DISCUSSION TITLE
How to Make Ghostty default term on mac?

### DISCUSSION THREAD

**Discussion by chertik77** [ghostty-org/ghostty#4082] [Q&A]
For example when I want to open external terminal via VsCode, it opens native terminal, i want him to open Ghostty, and completely replace my terminal with Ghostty

**Comment by rhodes-b** [comment]
open /usr/bin/ select an item, right click, select "Get Info"

![image](https://github.com/user-attachments/assets/a4cf5b1e-7588-4637-914b-cbe3be096045)

Select "Ghostty" and then click on "Change All..."

**Reply by chertik77** [reply]
I'm not sure if i've done it correct, but vscode still opens default terminal

**Reply by chertik77** [reply]
> open /usr/bin/ select an item, right click, select "Get Info"
> 
> ![image](https://private-user-images.githubusercontent.com/59537185/399322209-a4cf5b1e-7588-4637-914b-cbe3be096045.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU1NzM0NjMsIm5iZiI6MTczNTU3MzE2MywicGF0aCI6Ii81OTUzNzE4NS8zOTkzMjIyMDktYTRjZjViMWUtNzU4OC00NjM3LTkxNGItY2JlM2JlMDk2MDQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjMwVDE1MzkyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI1YTI5ODY0NTdkYzAxYmEwMjc3ZWU4MzZkOWRhODU1OTJlMTIzMmZmYzBkNTk2OGM2Yzg5MzFjNjM4YzFkMzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.vGCunoTWStGRX7AUcp3Pr0CQQCyP6f7Z0nVuWdxrOdg)
> 
> Select "Ghostty" and then click on "Change All..."

in /usr/bin/ i can select any item? if yes, i replaced it everywhere, even finder show me message that like you cannnot replace document only once, replace it everywhere? I clicked yes

**Reply by rhodes-b** [reply]
I dont think you can embed ghostty into vscode right now
